core/controllers/admin.py

```diff
@@ -297,13 +297,14 @@ def _create_dummy_question(
             state_domain.Solution(
                 'TextInput', False, 'Solution', state_domain.SubtitledHtml(
                     'solution', '<p>This is a solution.</p>')).to_dict())
+        solution = state_domain.Solution.from_dict('TextInput', solution_dict)
```

Comment:
Change solution_dict to solution and remove `to_dict()` from the end.

Reply:
sure, will do ,but   doubt in one ccomment.
https://github.com/oppia/oppia/pull/9431#discussion_r436286416

Reply:
okay